### PR TITLE
Activating log and processes collection by default + EU support

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Activates Log and processes collection by default for AWS ECS task description + EU support

### Motivation
Better product, easier config.

### CODE REVIEW

File: static/json/datadog-agent-ecs.json [hunk 1]
```diff
@@ -26,11 +36,23 @@
...
+        {
+          "name": "DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL",
+          "value": "true"
+        },
+        {
+          "name": "DD_PROCESS_AGENT_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "SD_BACKEND",
```
Comment (xvello) [inline static/json/datadog-agent-ecs.json]: this option is not used for agent6 anymore, we can remove it from all agent6 specs

File: static/json/datadog-agent-ecs-eu.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "datadog-agent",
+      "image": "datadog/agent:latest",
+      "cpu": 10,
+      "memory": 256,
```
Comment (xvello) [inline static/json/datadog-agent-ecs-eu.json]: too bad json does not support comments, I'd love to add a "customize according to observed usage". Is that something we could add to the doc that links to it?

File: static/json/datadog-agent-ecs1-eu.json [hunk 3]
```diff
@@ -42,8 +55,12 @@
           "value": "true"
         },
         {
-      	  "name": "SD_BACKEND",
-	  "value": "docker"
+          "name": "DD_PROCESS_AGENT_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "SD_BACKEND",
```
Comment (hkaj) [inline static/json/datadog-agent-ecs1-eu.json]: ditto

File: static/json/datadog-agent-ecs1.json [hunk 4]
```diff
@@ -26,11 +36,23 @@
...
+        {
+          "name": "DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL",
+          "value": "true"
+        },
+        {
+          "name": "DD_PROCESS_AGENT_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "SD_BACKEND",
```
Comment (hkaj) [inline static/json/datadog-agent-ecs1.json]: ditto

File: static/json/datadog-agent-ecs.json (hunk 1 above)
Comment (l0k0ms) [reply static/json/datadog-agent-ecs.json]: fixed :)

File: static/json/datadog-agent-ecs-eu.json (hunk 2 above)
Comment (l0k0ms) [reply static/json/datadog-agent-ecs-eu.json]: we could indeed, I'm going to add it.

**Comment by NBParis** [comment]
I think we should rename the PR as we do not enable them by default. We prepare all the configuration and let them enable everything with one single flag.

File: static/json/datadog-agent-ecs1-eu.json [hunk 5]
```diff
@@ -33,17 +38,25 @@
...
+          "value": "datadoghq.eu"
+        },
+        {
+          "name": "DD_LOGS_CONFIG_LOGS_DD_URL",
+          "value": "agent-intake.logs.datadoghq.eu:443"
+        },
         {
           "name": "DD_LOGS_ENABLED",
-          "value": "true"
+          "value": "false"
```
Comment (NBParis) [inline static/json/datadog-agent-ecs1-eu.json]: Well the `ecs-logs.json` should have log enabled.

File: static/json/datadog-agent-ecs1-eu.json [hunk 6]
```diff
@@ -33,17 +38,25 @@
           "name": "DD_API_KEY",
           "value": "<YOUR_API_KEY>"
         },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.eu"
+        },
+        {
+          "name": "DD_LOGS_CONFIG_LOGS_DD_URL",
+          "value": "agent-intake.logs.datadoghq.eu:443"
```
Comment (NBParis) [inline static/json/datadog-agent-ecs1-eu.json]: I'm not seeing the us version anymore. Am I missing it somewhere else?

Comment (irabinovitch) [reply static/json/datadog-agent-ecs1-eu.json]: Didn't 6.8 make this easier with a single SITE variable? or is that 6.9? Can we document based on that?  Theres far too many options here.

**Comment by irabinovitch** [comment]
Rather than having 4  json files can we just have 1 with instructions on how to update them for your needs?

File: static/json/datadog-agent-ecs1-eu.json (hunk 6 above)
Comment (l0k0ms) [reply static/json/datadog-agent-ecs1-eu.json]: The dd_site variable works with 6.8 but doesn't support the log endpoint yet. This support will come with 6.9. Hence the need to have it set up independently for now.

**Comment by l0k0ms** [comment]
@irabinovitch with 6.9 it will be easier since we could by default set dd_site to `datadoghq.com` and then just tell people to update the variable.

I think we should keep the Agent 5 configuration file since it's froozen

But I can remove the logic for amazon linux 1.0 and just tell people to update the volume path indeed.

What do you think?

**Pull request closed without merging**